data_architect_misc/data_transformer/configs/APAC_Taiwan/step1_transform_raw_investment_data.json

```diff
@@ -12,6 +12,7 @@
     "write_output": true,
     "output_folder_path": "./output/APAC_Taiwan/",
     "output_file_name_prefix": "Transformed_Taiwan_20200501_20200531_",
+    "output_encoding" : "utf-8-sig",
```

Comment:
Can we just leave it to "utf-8"? Does it work? If so, please leave it to "utf-8" because I'm afraid using "utf-8-sig" will create issues when we upload this file to SQL Server.

Reply:
Don Phyo, I tested it leaving only utf-8 and it didn't work, and then I tried to leave the utf-8-sing and it worked correctly

Reply:
Please leave a comment before "output_encoding" like below:
"__output_encoding_comment__": "In Taiwan input files, we have to use utf-8-sig as output encoding or else it does not work. The character outputs become incorrect if we use utf-8."